### PR TITLE
Moved Grid/Ruler button

### PR DESCRIPTION
**Pull request by a19steis**
Moved Grid/Ruler button from the options panel to the toolbox.

### CODE REVIEW

File: DuggaSys/diagram.php [hunk 1]
```diff
@@ -80,6 +80,15 @@
...
         </fieldset>
+        <fieldset>
+        <legend>Toggle</legend>
+        <div id="gridToggle" class="diagramIcons" onclick='toggleGrid()'>
+                    <img src="../Shared/icons/diagram_grid.svg"/>
+                </div>
+                <div id="rulerToggle" class="diagramIcons active" onclick='toggleRuler()'>
+                    <img src="../Shared/icons/diagram_ruler.svg"/>
+                </div>
+        </fieldset>
```
Comment (a19hammi) [inline DuggaSys/diagram.php:91]: Indentation isn't really the most beautiful. Please fix this. Apart from this everything is OK.

**Review by a19hammi — CHANGES_REQUESTED** [review]
Indentation incorrect, follow code standard.
Doesn't need follow-up review. Just fix and submit.

File: DuggaSys/diagram.php (hunk 1 above)
Comment (a19steis) [reply DuggaSys/diagram.php]: Fixed

**Review by a19hammi — APPROVED** [review]
LGTM!

**Pull request merged**